src/services/collector.ts

```diff
@@ -7,6 +7,7 @@ import dateHelper from "../utils/dateHelper";
 import fs from 'fs';
 import path from 'path';
 import {Pricer} from "./pricer";
+import {ignored} from "../index"
```

Comment:
why do you need to pass it like this ? Can't you use the path from the constructor, your withoutIgnored variable ?

Reply:
I'm not 100% sure what you mean; `ignored` is imported from index as the `config.getIgnored()` function is called there. The `ignored` const is then used to filter out the list of files and create the final `withoutIgnored`. Please could you elaborate?